lib/pspdfkit_widget.dart

```diff
@@ -0,0 +1,55 @@
+///
+///  Copyright © 2018-2019 PSPDFKit GmbH. All rights reserved.
+///
+///  THIS SOURCE CODE AND ANY ACCOMPANYING DOCUMENTATION ARE PROTECTED BY INTERNATIONAL COPYRIGHT LAW
+///  AND MAY NOT BE RESOLD OR REDISTRIBUTED. USAGE IS BOUND TO THE PSPDFKIT LICENSE AGREEMENT.
+///  UNAUTHORIZED REPRODUCTION OR DISTRIBUTION IS SUBJECT TO CIVIL AND CRIMINAL PENALTIES.
+///  This notice may not be removed from this file.
+///
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class PspdfWidget extends StatefulWidget {
```

Comment:
I would rename the class to `PspdfkitWidget`, it's more in line with our current naming convention and also follows the name of the file (`pspdfkit_widget.dart`).